pygmsh/opencascade/geometry.py

```diff
@@ -86,13 +86,30 @@ def add_wedge(self, *args, **kwargs):
         self._GMSH_CODE.append(p.code)
         return p
 
+    def volumes_inside(self, ex_vol, in_vols):
+        """ Function uses the boolean operations in order to make given internal volumes to 
+            belong to external one.
+
+            This function exists in order to make possible the assignment to the physical volumes
+            and boundary surfaces.
```

Comment:
I don't really understand what this wants to say. Internal, external? Perhaps a test would help clarifying things.

Reply:
This is too specific indeed. I have removed it. It only applies any boolean operation with delete_first=True and delete_other=False.